Orange/widgets/data/tests/test_owfile.py

```diff
@@ -176,10 +176,12 @@ def test_file_not_found(self):
         self.open_dataset("iris")
         self.assertFalse(self.widget.Error.file_not_found.is_shown())
 
+
```

Comment:
Lint doesn't complain, but PEP8 does. Could you please remove this line, since we are tidying up the files with this PR.

Reply:
Done. And pushed. Please be back when tests are finished. :)